demisto_sdk/commands/download/downloader.py

```diff
@@ -93,6 +92,29 @@
 }
 
 
+def map_uuid_to_name(
+    content_item_string: str, scripts_mapper: dict, file_suffix: str
+) -> dict:
+    """
+    Gets id from content item, if the id is UUID it maps it
+    to the content items' name.
+    Args:
+        content_item_string (str): content item as a string.
+        scripts_mapper (dict): a mapper from id to name for content items.
+        file_suffix (str): suffix of the file.
+    Returns:
+        dict: scripts_mapper
+    """
+    if file_suffix == ".yml":
+        content_item = yaml.load(content_item_string)
```

Comment:
does `load` work well on a list of strings? 🤔 (compared to `loads` of json)
check out `get_file` if you have the path

Reply:
As I understand it get_file will load the file content into memory and then we need to perform the yaml.load
However when checking your input I encountered yaml.safe_load() which is more secured as yaml.load() can also load arbitrary code.
Ill check with yaml.safe_load()

Reply:
note, this isn't `ruamel.yaml`, but our own `YAML_Handler`, so there may not be a `safe_load()`